### PR TITLE
added computer and department models

### PR DESCRIPTION
**Pull request by Downeastrussell**
# Description

Added models for Computer and Department

## Steps to Test
Compare Models to ERD

(x) I have commented my code, particularly in hard-to-understand areas
(x)  I have made corresponding changes to the documentation
(x)  My changes generate no new errors
(x)  I have added tests that prove my fix is effective or that my feature works
(x)  New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by BelcherCharles** [comment]
:hot_pepper

**Comment by CreativeFitz** [comment]
👍

File: BangazonAPI/BangazonAPI/Models/Department.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+{
+    public class Department
+    {
+        public int Id { get; set; }
+
+        public int Budget { get; set; }
+
+        public string Name { get; set; }
+    }
+}
```
Comment (kimpreece) [inline BangazonAPI/BangazonAPI/Models/Department.cs:18]: Ticket #7 says that "If the query string parameter of ?_include=employees is provided, then all employees in the department(s) should be included in the response."  Does the Model allow for this?

**Review by kimpreece — CHANGES_REQUESTED** [review]

File: BangazonAPI/BangazonAPI/Models/Department.cs (hunk 1 above)
Comment (Downeastrussell) [reply BangazonAPI/BangazonAPI/Models/Department.cs:18]: I added a List of <Employee> to the Department Model

**Review by kimpreece — CHANGES_REQUESTED** [review]
Looks good but is there are build errors because employee model doesn't exist yet

**Review by kimpreece — CHANGES_REQUESTED** [review]
Will you add a checklist from the template - 
https://github.com/NewForce-at-Mountwest/bangazon-api-bucatini/blob/master/.github/pull_request_template.md

**Review by kimpreece — APPROVED** [review]

**Pull request merged**